{
  "uuid": "9e2ca5f5-7489-48fb-9320-7b26bae5e266",
  "historyId": "1263c6d42f8f622522086e88c8dd3598",
  "fullName": "EndToEndTest.EndToEndTest.testLoginWithInvalidCredentials",
  "labels": [
    {
      "name": "package",
      "value": "EndToEndTest.EndToEndTest"
    },
    {
      "name": "testClass",
      "value": "EndToEndTest.EndToEndTest"
    },
    {
      "name": "testMethod",
      "value": "testLoginWithInvalidCredentials"
    },
    {
      "name": "parentSuite",
      "value": "Default Suite"
    },
    {
      "name": "suite",
      "value": "ProjectSourceLocation.java"
    },
    {
      "name": "subSuite",
      "value": "EndToEndTest.EndToEndTest"
    },
    {
      "name": "host",
      "value": "Mohamed"
    },
    {
      "name": "thread",
      "value": "7928@Mohamed.main(1)"
    },
    {
      "name": "framework",
      "value": "testng"
    },
    {
      "name": "language",
      "value": "java"
    },
    {
      "name": "owner",
      "value": "Mohamed Nasser"
    },
    {
      "name": "epic",
      "value": "Automation Exercise"
    },
    {
      "name": "severity",
      "value": "critical"
    }
  ],
  "links": [
    {
      "name": "https://www.automationexercise.com/login",
      "type": "custom"
    }
  ],
  "name": "testLoginWithInvalidCredentials",
  "status": "skipped",
  "statusDetails": {
    "known": false,
    "muted": false,
    "flaky": false,
    "message": "Method EndToEndTest.EndToEndTest.testLoginWithInvalidCredentials() on instance EndToEndTest.EndToEndTest@7e094740 depends on not successfully finished methods \n[EndToEndTest.EndToEndTest.testLoginWithValidCredentials() on instance EndToEndTest.EndToEndTest@7e094740]",
    "trace": "java.lang.Throwable: Method EndToEndTest.EndToEndTest.testLoginWithInvalidCredentials() on instance EndToEndTest.EndToEndTest@7e094740 depends on not successfully finished methods \n[EndToEndTest.EndToEndTest.testLoginWithValidCredentials() on instance EndToEndTest.EndToEndTest@7e094740]\r\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:159)\r\n\tat org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)\r\n\tat org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)\r\n\tat java.base/java.util.ArrayList.forEach(ArrayList.java:1597)\r\n\tat org.testng.TestRunner.privateRun(TestRunner.java:739)\r\n\tat org.testng.TestRunner.run(TestRunner.java:614)\r\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:421)\r\n\tat org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)\r\n\tat org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)\r\n\tat org.testng.SuiteRunner.run(SuiteRunner.java:312)\r\n\tat org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)\r\n\tat org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)\r\n\tat org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)\r\n\tat org.testng.TestNG.runSuitesLocally(TestNG.java:1208)\r\n\tat org.testng.TestNG.runSuites(TestNG.java:1112)\r\n\tat org.testng.TestNG.run(TestNG.java:1079)\r\n\tat com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:65)\r\n\tat com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:105)\r\n"
  },
  "stage": "finished",
  "description": "Verify system shows an error message when logging in with invalid credentials...",
  "steps": [],
  "attachments": [],
  "parameters": [],
  "start": 1735444190557,
  "stop": 1735444190561
}